certs/main.tf

```diff
@@ -1,15 +1,8 @@
-# openvpn_server
-
-provider "aws" {
```

Comment:
@blakeneyops ah that's nice, the provider doesn't have to be in the module itself. Looks like switching to role based access won't be so bad after all.